.github/workflows/build-push.yaml

```
@@ -0,0 +1,65 @@
+name: "Build & Push"
+
+on:
+  push:
+    branches:
+      - "develop"
+      - "main"
+    paths:
+      - "packages/**"
+
+jobs:
+  build_push_image:
+    name: ${{ matrix.image_name }}
+    runs-on: ubuntu-latest
+    environment: dev
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image_name: probing-api
+            dockerfile_path: packages/probing-api
+          - image_name: probing-caller
+            dockerfile_path: packages/probing-caller
+          - image_name: probing-eservice-operations
+            dockerfile_path: packages/probing-eservice-operations
+          - image_name: probing-eservice-registry-reader
+            dockerfile_path: packages/probing-eservice-registry-reader
+          - image_name: probing-eservice-registry-updater
+            dockerfile_path: packages/probing-eservice-registry-updater
+          - image_name: probing-response-updater
+            dockerfile_path: packages/probing-response-updater
+          - image_name: probing-scheduler
+            dockerfile_path: packages/probing-scheduler
+          - image_name: probing-statistics-api
+            dockerfile_path: packages/probing-statistics-api
+          - image_name: probing-telemetry-writer
+            dockerfile_path: packages/probing-telemetry-writer
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: normalize branch name for tagging
+        run: |
+          NORM_TAG=$(echo "$GITHUB_REF_NAME" | tr -s "/" "-")
+          echo "NORM_TAG=$NORM_TAG" >> $GITHUB_ENV
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push Docker image
+        id: build_push
+        uses: docker/build-push-action@v5.0.0
+        with:
+          context: .
+          file: ${{ matrix.dockerfile_path }}/Dockerfile
+          push: true
+          tags: |
+            ghcr.io/interop-be-${{ matrix.image_name }}:${{ env.NORM_TAG }}
```

```suggestion
            ghcr.io/interop-be-${{ matrix.image_name }}:${{ github.ref_name }}
```
Since we don't need tag normalization anymore